clang/docs/ClangFormatStyleOptions.rst

```diff
@@ -3220,6 +3220,25 @@ the configuration (without a prefix: ``Auto``).
 
 
 
+.. _EmptyLinesAfterIncludes:
+
+**EmptyLinesAfterIncludes** (``Unsigned``) :versionbadge:`clang-format 18` :ref:`¶ <EmptyLinesAfterIncludes>`
```

Comment:
18 will be correct today, but it won't be tomorrow (as they branch I think), I doubt we are going to land this in 18

Reply:
Where do I have to sign up for that knowledge? Back with the mailing lists I got that information, but for this new stuff I'm kind of too old.

Reply:
I've always looked at the "Upcoming Release" over at https://llvm.org/